client/assets/stylesheets/shared/mixins/_hide-content-accessibly.scss

```diff
@@ -2,12 +2,17 @@
 // ==========================================================================
 // Hide content accessibly mixin
 // Hides content in a way that makes it still accessible to screen readers
+// Based on https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/
```

Comment:
Any idea why we didn't just copy/paste the CSS from the accessibility handbook?

Reply:
I'm guessing that some of the styles aren't required for our supported browsers. Can you shed any light on this @jsnajdr?

Reply:
Updated styles to match the accessibility handbook in 3d920d7e05315eadbff64f6104ca4ade24939c28.

Reply:
This mixin was created in 2015 and hasn't changed in the 6 years that elapsed since then. Updating the CSS to conform to the current a11y handbook sounds like a good idea. None of the styles there look outdated. Maybe they're covering some edge cases we're not aware of.

Reply:
Thanks for the extra context and history there!